rest_server/src/models/model_init_handler.js

```diff
@@ -48,9 +48,27 @@ const createTemplates = async models => {
   );
 };
 
+const createStorageBlobs = async models => {
+  try {
+    const envBlob = JSON.parse(process.env.AZURE_STORAGE);
+    const defaultBlob = {
+      type: envBlob.type,
+      storageAccount: envBlob.storage_account,
+      containerName: 'marketplace',
+      connectionStrings: envBlob.connection_strings,
+      tokens: [''],
+      users: [''],
+    };
```

Comment:
what if the process.env.AZURE_STORAGE is null?

Reply:
It seems json.parse will throw an error "Thrown: SyntaxError: Unexpected token u in JSON at position 0", may be there should be a try catch ?

Reply:
ok, an overall try catch is enough, no need to add more logics